src/dialogs/more-info-dialog.html

```diff
@@ -16,6 +16,11 @@
     <style>
       paper-dialog {
         font-size: 14px;
+        width: 450px;
+      }
+
+      paper-dialog[data-domain=camera] {
+        min-width: auto;
```

Comment:
Disregard my other comment. I see what you were saying. I changed camera to width: auto. I can add a size restriction for that too but I don't think it is necessary. I think we should let that float to whatever size the camera presents. Let me know.